base/features/kotlin/feature.yml

```diff
@@ -17,4 +17,6 @@ dependencies:
   - scope: kapt
     coords: io.micronaut:inject-java
   - scope: kaptTest
-    coords: io.micronaut:inject-java
\ No newline at end of file
+    coords: io.micronaut:inject-java
+  - scope: runtime
+    coords: com.fasterxml.jackson.module:jackson-module-kotlin:2.9.+
```

Comment:
I think it is better to specify the patch version here

Reply:
sure thing, done